onboarder/installation/index.rst

```diff
@@ -166,4 +166,4 @@ A new Elasticsearch index is needed for pre ref products. In order to create it,
 Setup synchronization
 ---------------------
 
-Now that you have a working Onboarder bundle, you have to setup :doc:`synchronization </onboarder/synchronization/index>`
\ No newline at end of file
+Now that you have a working Onboarder bundle, you have to setup :doc:`synchronization </onboarder/synchronization/index>`
```

Comment:
What is the diff on this line? :thinking:

Reply:
No idea